src/components/global/TaskBoardSampleData.js

```diff
@@ -0,0 +1,217 @@
+const TaskBoardSampleData = {
```

Comment:
do we need sample data now that we connected to firebase?

Reply:
^^

Reply:
Not connected yet